packages/code-assist/README.md

```diff
@@ -330,6 +330,150 @@ This server supports two standard MCP communication protocols:
 
 -----
 
+<!-- [START maps_StreamableHTTPGuide] -->
+
+## 🌐 StreamableHTTP Developer Guide
```

Comment:
s/StreamableHTTP/Streamable HTTP/ 
need a space in between for all instances except code